tests/nvme_smart_log_test.py

```diff
@@ -83,4 +83,6 @@ def get_smart_log_all_ns(self):
     def test_smart_log(self):
         """ Testcase main """
         assert_equal(self.get_smart_log_ctrl(), 0)
-        assert_equal(self.get_smart_log_all_ns(), 0)
+        smlp = self.supp_check_id_ctrl("lpa")
```

Comment:
Please introduce supp_check_id_ctrl first in the patch series.

Reply:
Reorder the patch series as per the ask. Thanks @igaw